string need_name    		# 需求的名称
int8 satisfy_value
